### ISSUE TITLE
Something happen

### ISSUE THREAD

**Issue by robi09** [avocode/avocode#731]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.5.3
Source UID: 27627
```
Error: invalid stored block lengths
  at Zlib._handle.onerror (zlib.js:345:17)

```

**Comment by robi09** [comment]
I was uploading a PSD file into Avocode Windows APP. After the processing was completed this error comes up.

**Comment by petrbrzek** [comment]
Hi, we've released 2.5.4 that should fix this issue. If you still have this problem in the new app, feel free to report it as a new issue
